cli/integrity/integrity.go

```diff
@@ -4,17 +4,23 @@ import (
 	"errors"
 
 	"github.com/spf13/pflag"
-
-	"github.com/tarantool/tt/cli/cluster"
 )
 
 var (
-	ErrNotConfigured = errors.New("integration check is not configured")
+	// ErrNotConfigured is reported when integrity check is not configured
+	// in the command context.
+	ErrNotConfigured = errors.New("integrity check is not configured")
 )
 
-var FileRepository Repository = dummyRepository{}
+// IntegrityCtx is context required for integrity checks.
+type IntegrityCtx struct {
+	// Repository is a repository used to check integrity of files.
+	Repository Repository
+}
 
-var HashesName = ""
+// HashesName is a name of a file containing file hashes that
+// require checking.
+const HashesName = ""
 
 // Signer implements high-level API for package signing.
 type Signer interface {
```

Comment:
Why do we need the `Signer` in the public API?

Reply:
`Signer` is used in `pack`